src/OrleansAzureUtils/Storage/OrleansSiloInstanceManager.cs

```diff
@@ -241,21 +241,28 @@ private static Uri ToGatewayUri(SiloInstanceTableEntry gateway)
 
         private IEnumerable<SiloInstanceTableEntry> FindAllGatewaySilos()
         {
-            logger.Info(ErrorCode.Runtime_Error_100277, "Searching for active gateway silos for deployment {0} ...", this.DeploymentId);
+            if (logger.IsVerbose) logger.Verbose(ErrorCode.Runtime_Error_100277, "Searching for active gateway silos for deployment {0}.", this.DeploymentId);
```

Comment:
Why do you want to downgrade this message to Verbose? Wouldn't it be useful to see when we started and compare it to the "found" or "error" message after that?